models.py

```diff
@@ -363,6 +363,15 @@ def get_shipping_samples(self, limit=None, update_cache=False):
       q.order('name')
       features = q.fetch(None)
 
+      # Get all experimental framework features. Ordered by shipping milestone (latest first).
+      q = Feature.all()
+      q.filter('impl_status_chrome =', EXPERIMENTAL_FRAMEWORK)
+      q.order('-impl_status_chrome')
+      q.order('-shipped_milestone')
+      q.order('name')
+      experimental_features = q.fetch(None)
+      features.extend(experimental_features)
```

Comment:
this will put all the experimental features at the end of the list instead of interweaving them. Is that what you want?